en/docs/Administer/ProductSecurity/enabling-hostname-verification.md

```diff
@@ -1,97 +1,41 @@
-        -Dorg.wso2.ignoreHostnameVerification=true \
-        -Dhttpclient.hostnameVerifier=AllowAll \
-    ```
+### Configuring hostname verification
 
-    If you are disabling hostname verification for **WSO2 EI 6.1.1** , you need to use both system properties listed below.
+The hostname verification is **disabled** by default. This is done using the `org.opensaml.httpclient.https.disableHostnameVerification` and `httpclient.hostnameVerifier` properties in the product's startup script ( `wso2server.sh` for Linux and `wso2server.bat` for Windows) as shown below. The product startup script is stored in the `<PRODUCT_HOME>/bin` directory. This property will be effective during server startup.
 
-    ``` java
-            -Dhttpclient.hostnameVerifier=AllowAll \
-            -Dorg.opensaml.httpclient.https.disableHostnameVerification=true \
-    ```
-
-    If you are disabling hostname verification for **WSO2 IS** , use the following system properties.
-
-    ``` java
-            -Dorg.opensaml.httpclient.https.disableHostnameVerification=true \
-            -Dhttpclient.hostnameVerifier="AllowAll" \
-    ```
+```  
+    -Dorg.opensaml.httpclient.https.disableHostnameVerification=true \
+    -Dhttpclient.hostnameVerifier=AllowAll \
+```
 
+!!! note
+    The values you can use with this `httpclient.hostnameVerifier` property are explained below. Note that these values will behave the same as synapse hostname verification options.
+    
+    -   **DefaultAndLocalhost:** This is the value that is enabled, by default. This means that all hostnames, except the ones listed below, will be verified against the hostnames specified in the product's SSL certificate. That is, the following hostnames will be allowed regardless of the server's certificate.
+    
+        -   localhost
+    
+        -   localhost.localdomain
+    
+        -   127.0.0.1
+    
+        -   ::1
+    
+        Note that if the wildcard symbol is used to specify a hostname in the SSL certificate (such as \*. [foo.com](http://foo.com/) ), all the subdomains of \*. [foo.com](http://foo.com/) are also included. That is a hostname that matches a subdomain of \*. [foo.com](http://foo.com/) will also be allowed for access.
+    
+    -   **Strict:** When this mode is enabled, hostnames will be strictly verified against the hostname specified in the product's SSL certificate. For example, if "\*. [foo.com](http://foo.com/) " is specified as the hostname in the certificate, only the hostnames at the same level will be authorized by the server. That is subdomains such as " [a.b.foo.com](http://a.b.foo.com/) " will **not** be authorized.
+    
+    -   **AllowAll:** This option turns off hostname verification for the server. Note that this is not recommended in a production setup and should only be used for demonstrations and testing.
+
+!!! note 
+    Refer [Enabling HostName Verification](https://docs.wso2.com/display/ADMIN44x/Enabling+HostName+Verification) for more information.
```

Comment:
Link needs to be updated

Reply:
Thanks. Fixed. https://github.com/wso2/docs-apim/pull/586/commits/cd0d07b18330b9b7af91f6421cde896218160865